gadfly/gap_fill_generator.py

```diff
@@ -30,15 +30,28 @@ def find_named_entities(self):
         return entities
 
     def summarize_sentences(self):
```

Comment:
Can you refactor this to follow the pattern that I used for the blank_types? Look at the generate_questions method in the development branch now.